src/core/config.py

```diff
@@ -1,15 +1,14 @@
 import os
-import google.generativeai as Client
 from github import Github
-from typing import Tuple
 
 class Config:
     GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
     GEMINI_API_KEY = os.environ.get('GEMINI_API_KEY')
     GEMINI_MODEL = os.environ.get('GEMINI_MODEL', 'gemini-1.5-flash-002')
+    OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+    OPENAI_MODEL = os.environ.get('OPENAI_MODEL', 'gpt-4o-mini')
```

Comment:
While adding OpenAI support is good, there's no actual usage of `OPENAI_API_KEY` or `OPENAI_MODEL` in `initialize_clients`.  This should be addressed to ensure the added configuration is functional.